src/atci.c

```diff
@@ -228,6 +228,36 @@ bool atci_param_get_uint(atci_param_t *param, uint32_t *value)
     return true;
 }
 
+bool atci_param_get_int(atci_param_t *param, int32_t *value)
+{
+    if (param->offset >= param->length)
+    {
+        return false;
+    }
+
+    char c;
+
+    *value = 0;
```

Comment:
No need to perform the initialization here. atci_param_get_uint does that internally.